README.md

```diff
@@ -95,7 +102,51 @@ Make sure to start from a brand-new environment to avoid conflicts with previous
 
 1. In App Launcher, select the **Dreamhouse** app.
 
-## Sample Data Import
+## Installing Dreamhouse using a Developer Edition Org or a Trailhead Playground
+
+Follow this set of instructions if you want to deploy the app to a more permanent environment than a Scratch org.
+This includes non source-tracked orgs such as a [free Developer Edition Org](https://developer.salesforce.com/signup) or a [Trailhead Playground](https://trailhead.salesforce.com/).
+
+Make sure to start from a brand-new environment to avoid conflicts with previous work you may have done.
+
+1. Authenticate with your Trailhead Playground or Developer org and provide it with an alias (**mydevorg** in the command below):
+
+    ```
+    sfdx force:auth:web:login -d -a mydevorg
+    ```
+
+1. Clone this repository:
+
+    ```
+    git clone https://github.com/dreamhouseapp/dreamhouse-lwc
+    cd dreamhouse-lwc
+    ```
+
+1. If you are setting up a Developer Edition: go to **Setup**, under **My Domain**, [register a My Domain](https://help.salesforce.com/articleView?id=domain_name_setup.htm&type=5).
+
+1. Run this command in a terminal to deploy the app.
+
+    ```
+    sfdx force:source:deploy -p force-app
+    ```
+
+1. Assign the `dreamhouse` permission set to the default user.
+
+    ```
+    sfdx force:user:permset:assign -n dreamhouse
+    ```
+
+1. Import some sample data.
+
+    ```
+    sfdx force:data:tree:import -p ./data/sample-data-plan.json
```

Comment:
For some unknown reason this sample app doesn't use the same name for sample data plan file (it's missing the `-plan` part). We should unify those names across sample apps.
Can you do the following:
- rename the file to `sample-data-plan.json`
- update the `bin/*.sh` files
- update the other reference to data import in the readme (in scratch org setup).